tests/cross_functional/ui/test_create_pool_block_pool.py

```diff
@@ -118,6 +122,19 @@ def test_create_delete_pool(
         # Getting IO results
         get_fio_rw_iops(self.pod_obj)
 
+        # Checking the raw capcity is loaded on the UI or not.
+        blockpool_ui_object = BlockPoolUI()
+        if not blockpool_ui_object.pool_raw_capacity_loaded(self.pool_name):
+            raise BlockPoolRawCapacityNotLoaded(
```

Comment:
Shivam, I am not quite agree that these should bring errors/exceptions. 
I thing it would be better to throw exception here, to increase number of Fails but not Errors in report.

@petr-balogh since you're triggering regression, what do you think?

Reply:
updated